src/main/resources/db/migration/V1676213985__user_permissions_table.sql

```diff
@@ -202,8 +222,43 @@ VALUES ('ROLE_ADMIN', 'This is Administrator role. This is the highest level Rol
 INSERT INTO `user`(user_name, first_name, last_name, email, password, mobile, is_active, description, role_id, created_by, modified_by)
 VALUES ('admin', 'Admin', 'Admin', 'kapas_2023@gmail.com', 'yq2Dq6hosT3V9tJFWMNiwA==', '999-999-9999', 1, 'This is Admin User.', 1, 1, 1);
 
-
-
+INSERT INTO `workflow`(workflow_id, description, version, workflow_json, created_by, modified_by)
+VALUES ('SHRI_BALAJI_AGR_PROCUREMENT_PROCESS', 'This workflow is for the digital automation of the currently followed procuerement process at Shri Balaji Agro Pvt Ltd.', 1,
+"{
+    'workflow_id':'SHRI_BALAJI_AGR_PROCUREMENT_PROCESS',
+    'total_task':4,
+    'task':[
+        {
+            'task_number':1,
+            'task_id':'GENERATE_TOKEN',
+            'task_name':'Generate Token',
+            'description':'',
+            'required':'true'
```

Comment:
required should be boolean as true and not 'true'

Reply:
done